content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -242,6 +242,12 @@ message ContainerDevices {
 GetAllocatableResources provides information on resources initially available on the worker node.
 It provides more information than kubelet exports to APIServer.
 
+NOTE:
+
+- `GetAllocatableResources` should only be used to evaluate [allocatable](https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#node-allocatable) resources on a node. If the goal is to evaluate free/unallocated resources it should 
+be used in conjunction with the List() endpoint. The result obtained by `GetAllocatableResources` would remain the same unless the underlying resources exposed to kubelet change. This happens rarely but when it does (e.g. CPUs onlined/offlined, devices added/removed), client is expected to call `GetAlloctableResources` endpoint.
```

Comment:
Please wrap the long lines because 1) long lines make it harder to comment 2) long lines are difficult for change tracking. However, we always keep links on a single line for validation's purpose, i.e. don't break `[allocatable](link target ...)` into multiple lines.

We use relative links in the docs, so please remove 'https://kubernetes.io' from the link target.

The `List() endpoint` is confusing. Is it referring to the GRPC interface function?

Reply:
I would also use a callout shortcode here: https://kubernetes.io/docs/contribute/style/style-guide/#note

Reply:
> Please wrap the long lines because 1) long lines make it harder to comment 2) long lines are difficult for change tracking. However, we always keep links on a single line for validation's purpose, i.e. don't break [allocatable](link target ...) into multiple lines.

Sure, I will keep the above in mind.
> We use relative links in the docs, so please remove 'https://kubernetes.io' from the link target.

Ack
> The List() endpoint is confusing. Is it referring to the GRPC interface function?

Yes @tengqm, we are referring to the List GRPC interface function